### PR TITLE
[HPRO-537] Add participant origination information to WQ & participant summary pages

### PR DESCRIPTION
**Pull request by shyamnaru**

### CODE REVIEW

File: web/local-router.php [hunk 1]
```diff
@@ -2,10 +2,9 @@
 if (preg_match('#^/assets/#', $_SERVER["REQUEST_URI"])) {
     // always pass through for static assets
     return false;
-} elseif (preg_match('/\\.pdf$/', $_SERVER["REQUEST_URI"])) {
-    // for non-asset PDF URLs, explicitly send to index.php controller to avoid
+} elseif (preg_match('/\\.pdf|csv|json$/', $_SERVER["REQUEST_URI"])) {
+    // for non-asset PDF, CSV and JSON URLs, explicitly send to index.php controller to avoid
```
Comment (stephenyeargin) [inline web/local-router.php]: Guessing this is riding along with this change, but not specifically related.

Comment (shyamnaru) [reply web/local-router.php]: Yes, not specifically related but need to change this in order to test csv exports in localhost.

File: web/assets/js/views/workqueue.js [hunk 2]
```diff
@@ -69,6 +70,7 @@ $(document).ready(function() {
       { name: 'biobankId', visible: false, data: 'biobankId' },
       { name: 'language', visible: false, data: 'language', orderable: false  },
       { name: 'participantStatus', data: 'participantStatus' },
+      { name: 'participantOrigin', data: 'participantOrigin', visible: !!isDvType },
```
Comment (jt2k) [inline web/assets/js/views/workqueue.js]: Do we need to update the default order column index below?

**Comment by shyamnaru** [comment]
@jt2k Ready for review

**Pull request merged**